plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/githubPullRequest.ts

```diff
@@ -202,6 +203,11 @@ export const createPublishGithubPullRequestAction = (
             description:
               'The teams that will be added as reviewers to the pull request',
           },
+          commitMessage: {
+            type: 'string',
+            title: 'Commit Message',
+            description: 'The commit message for the PR commit',
```

Comment:
To match the rest of the schema

```suggestion
            description: 'The commit message for the pull request commit',
```